app/controllers/junior_stories_controller.rb

```diff
@@ -33,4 +37,41 @@ def junior_story_params
       :remote, :tech_team_size, :company_size, :company_age, :person_of_colour, :other,
       :publishing_consent, :freelancer)
   end
+
+  def allowed_params
+    params.permit(:job, :happy_in_job, :happy_info,
+      :gender, :city, :country, :days_per_week, :salary, :currency, :technology, :focus,
+      :age, :years_working_in_total, :years_working_at_job, :education, :first_job,
+      :remote, :tech_team_size, :company_size, :company_age, :person_of_colour, :other,
+      :publishing_consent, :freelancer)
+  end
+
+  # TODO: shouldn't hard code this, but I like having demo search terms in the list
+  def fields
```

Comment:
I don't think this really truly belongs anywhere^^ What makes you say it's better suited for the model? Thin controllers - fat models?